benchmarks/torchbench_model.py

```diff
@@ -244,16 +245,18 @@ def set_up(self):
     del benchmark
     self._cleanup()
 
-  def load_benchmark(self):
+  @functools.lru_cache(maxsize=1)
```

Comment:
I am afraid this doesn't solve the issue. `lru_cache` may not work across processes. We will still call the function later as a subprocess.  Have you tried this in TPU device?